packages/client/src/containers/FavoritesPage/FavoritesPage.container.js

```diff
@@ -3,16 +3,26 @@ import SearchInput from '../../components/SearchInput/SearchInput.component';
 import { getLocalStorage, setLocalStorage } from '../../utils/storageHelpers';
 import ProductCard from '../../components/ProductCard/ProductCard.component';
 import './FavoritesPage.style.css';
+import Modal from '../../components/Modal/Modal.component';
+import { ViewPageButton } from '../../components/ViewPageButton/ViewPageButton.component';
 
 export const FavoritesPage = () => {
+  const [openModal, setOpenModal] = useState({
+    modalStatus: false,
+    favoriteId: -1,
+  });
+  const toggleModal = () => {
+    setOpenModal({ modalStatus: false, favoriteId: -1 });
+    document.body.style.overflow = 'visible';
```

Comment:
In general you should never modify the CSS of the page directly, instead you should alter this with a CSS class that is added within the React markup instead.

Reply:
True I can remove it